.changeset/fair-meals-reflect.md

```diff
@@ -4,4 +4,6 @@
 
 Makes aggregation types nullable, even if the original property is non-nullable.
 
-This is because, in case of no nodes existing in the database, a null value will be returned by the aggregation
+This is because, in case of no nodes existing in the database, a null value will be returned by the aggregation.
+
+The old `*AggregateSelectioNonNullable`-types and `*AggregateSelectioNullable`-types are now merged into `*AggregateSelectio`-types`.
```

Comment:
Instead of changing the existing changeset, please make a new one. This is to make sure that the resulting changelog points to the correct PRs